src/alert-prelude.c

```diff
@@ -744,14 +763,23 @@ static void PacketToDataProtoTLS(const Packet *p, const PacketAlert *pa, idmef_a
 static void PacketToDataProtoSSH(const Packet *p, const PacketAlert *pa, idmef_alert_t *alert)
 {
     json_t *js, *s_js;
+    json_error_t error;
+    JsonBuilder *jsb;
+
     void *ssh_state = FlowGetAppState(p->flow);
 
     if (ssh_state == NULL)
         return;
 
+    jsb = jb_new_object();
+
     void *tx_ptr = rs_ssh_state_get_tx(ssh_state, 0);
     BUG_ON(tx_ptr == NULL);
-    js = rs_ssh_log_json(tx_ptr);
+    rs_ssh_log_json(tx_ptr, jsb);
```

Comment:
As @victorjulien mentioned about error checking, rs_ssh_log_json is one method that could fail. So should be checked.